components/bl-granim-component/README.md

```diff
@@ -0,0 +1,97 @@
+| colors                  | *JSON*                                                                                                                                                                                                                    | `[]`          |       | NO           | YES        | Specifies a JSON array containing the colors for the gradient. Watch [Codeless Examples](#Examples). |
+| imageSource             | *Text*                                                                                                                                                                                                                    |               |       | NO           | YES        | The source of the image(URL).                                                                         |
+| imagePosX               | *Select* <br> [`center`, `left`, `right`]                                                                                                                                                                                 | `center`      |       | NO           | YES        | The x-position of the image in the canvas.                                                            |
+| imagePosY               | *Select* <br> [`center`, `top`, `bottom`]                                                                                                                                                                                 | `center`      |       | NO           | YES        | The y-position of the image in the canvas.                                                            |
+| stretchModeX            | *Select* <br> [`none`, `stretch`, `stretch-if-smaller`, `stretch-if-bigger`]                                                                                                                                              | `none`        |       | NO           | YES        | Stretching the image horizontally.                                                                    |
+| stretchModeY            | *Select* <br> [`none`, `stretch`, `stretch-if-smaller`, `stretch-if-bigger`]                                                                                                                                              | `none`        |       | NO           | YES        | Stretching the image vertically.                                                                      |
+| blendingMode            | *Select* <br> [`multiply`, `screen`, `normal`, `overlay`, `darken`, `lighten`, `lighter`, `color-dodge`, `color-burn`, `hard-light`, `soft-light`, `difference`, `exclusion`, `hue`, `saturation`, `color`, `luminosity`] | `multiply`    |       | NO           | YES        | Options for blending images with a gradient.                                                          |
```

Comment:
Also use the readable names for properties in readme

blendingMode -> Blending Mode